### PR TITLE
test 1

### PR DESCRIPTION
**Pull request by ihtsham**

### CODE REVIEW

File: SprintTestPlan/readme.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-BTH
\ No newline at end of file
+BTH
+Test reveiw
```
Comment (nasirminhas) [inline SprintTestPlan/readme.txt:2]: More description is needed.

File: SprintTestPlan/readme.txt [hunk 2]
```diff
@@ -1 +1,2 @@
-BTH
\ No newline at end of file
+BTH
```
Comment (nasirminhas) [inline SprintTestPlan/readme.txt:1]: Use complete name!

**Review by nasirminhas — CHANGES_REQUESTED** [review]

File: SprintTestPlan/readme.txt (hunk 1 above)
Comment (ihtsham) [reply SprintTestPlan/readme.txt]: it has been resolve

**Comment by nasirminhas** [comment]
I am okay with the new changes

**Pull request merged**